Jenkinsfile

```diff
@@ -10,4 +10,7 @@ node {
   govuk.buildProject(
     brakeman: true,
   )
+
+  // Run against the MySQL 8 Docker instance on GOV.UK CI
+  govuk.setEnvar("TEST_DATABASE_URL", "mysql2://root:root@127.0.0.1:33068/signonotron2_test")
```

Comment:
Odd database name 😂 `signonotron2` doesn't seem like a very meaningful name.
But 👍🏻 for being consistent with what's already used in `database.yml`

Reply:
Yea haha. Very whimsical